plugins/processHooks/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const joi = require('joi');
+const { startHookEvent } = require('../webhooks/helper');
+
+const DEFAULT_MAX_BYTES = 1048576;
+
+/**
+ * Process Hooks API Plugin
+ * - Start pipeline events from scm webhook config
+ * @method register
+ * @param  {Hapi}       server  Hapi Server
+ * @param  {Object}     options Configuration
+ * @param  {Function}   next    Function to call when done
+ */
+const processHooksPlugin = {
+    name: 'processHooks',
+    async register(server, options) {
+        server.route({
+            method: 'POST',
+            path: '/processHooks',
+            options: {
+                description: 'Handle process hook events',
+                notes: 'Acts on pull request, pushes, comments, etc.',
+                tags: ['api', 'processHook'],
```

Comment:
need auth
https://github.com/sonic-screwdriver-cd/queue-service/blob/2bc193c5aaf246f25dbc8769fef92f9359420fb9/plugins/worker/lib/jobs.js#L286

Reply:
I fixed it.